CHANGES.md

```diff
@@ -131,6 +131,9 @@ Unreleased
   way to rerun all actions failed due to errors without restarting the build,
   e.g. via a Dune RPC call. (#4522, Andrey Mokhov)
 
+- Remove `dune compute`. It was broken and unused (#....,
```

Comment:
The PR number still needs to be filled in.

Reply:
```suggestion
- Remove `dune compute`. It was broken and unused (#4540,
```